static/scss/layout.scss

```diff
@@ -1,8 +1,34 @@
 // Layout goes here
+@import "node_modules/bootstrap/scss/bootstrap";
 
-// Fake style to confirm that webpack dev server works
-h1 {
-  border-radius: 25px;
-  border: 2px solid blue;
-  padding: 2px;
+$max-width: 1447px;  // arbitrary value, copied from ocw-next
+
+.site-wide-header {
```

Comment:
I think it's almost certain we will want a separate page of styles for the site detail page rather than putting them all in a stylesheet with styles for the entire site. I'd be in favor of doing that now rather than later